ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -35,6 +35,7 @@
 
     @Before
     public void setUp() throws Exception {
+        
```

Comment:
Any reason for adding this empty line `LN-38` ?

Reply:
Thanks , this was simply for triggering the build , Resolved